tensorflow/core/util/rocm_solvers.cc

```diff
@@ -111,7 +111,7 @@ struct ROCmSolverHandles {
   ~ROCmSolverHandles() {
     CHECK(wrap::rocblas_destroy_handle(parent_, rocm_blas_handle) ==
           rocblas_status_success)
-        << "Failed to destroy cuBlas instance.";
+        << "Failed to destroy rocBlas instance.";
```

Comment:
good catch, but keep this in a separate commit. 

Keeping commits organized by functionality, helps "cherry-pick" them easier